recipes/hostess/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "hostess" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hostess-{{ version }}.tar.gz
+  sha256: 319916189bc0cb8cf2c85cc2479f5e90cffcc858774a0734856c1365e3cdc912
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<39]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
```suggestion
    - python >=3.9
    - pip
  run:
    - python >=3.9
```

Reply:
That fixes the noarch lint you had. You should make it noarch and add the min python here.